### PR TITLE
Add Vector for forwarding Pipelines log

### PR DESCRIPTION
**Pull request by khrm**
This is needed to enable a newer version of Results which is shipped to other customers.

For now, we are using play.min.io for dev environment.

### CODE REVIEW

**Review by khrm — COMMENTED** [review]
/assign @enarha

File: hack/secret-creator/create-plnsvc-secrets.sh [hunk 1]
```diff
@@ -35,34 +45,12 @@ create_s3_secret() {
...
-type: Opaque
-stringData:
-  config.env: |-
-    export MINIO_ROOT_USER="$USER"
-    export MINIO_ROOT_PASSWORD="$PASS"
-    export MINIO_STORAGE_CLASS_STANDARD="EC:1"
-    export MINIO_BROWSER="on"
-EOF
+      --from-literal=aws_access_key_id="Q3AM3UQ867SPQQA43P2F" \
+      --from-literal=aws_secret_access_key="zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG" \
```
Comment (Roming22) [inline hack/secret-creator/create-plnsvc-secrets.sh:50]: I would not recommend exposing a token. If it is removed from the PR, it should be rotated.

Comment (khrm) [reply hack/secret-creator/create-plnsvc-secrets.sh:50]: This is a public instance of min.io for testing purpose. It's already expose by minio.

File: components/pipeline-service/development/kustomization.yaml [hunk 2]
```diff
@@ -8,55 +8,6 @@ commonAnnotations:
   argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
 
 resources:
-  - github.com/minio/operator?ref=v5.0.15
```
Comment (enarha) [inline components/pipeline-service/development/kustomization.yaml:11]: I remember this was recently changed by @amisstea , so I wonder if his team has any use of it. @amisstea please confirm we can remove that.

File: hack/secret-creator/create-plnsvc-secrets.sh (hunk 1 above)
Comment (enarha) [reply hack/secret-creator/create-plnsvc-secrets.sh:52]: I was skeptical as well, but those credentials are publicly available here https://min.io/docs/minio/linux/administration/minio-console.html#logging-in , and that's considered a temporary solution, so I'm OK with it.

**Pull request merged**